doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1272 @@
+      You can read more about the new command in the `nRF Util documentation <sdk-manager command_>`_.
+
+  * Mentions of commands that use tools from the nRF Command Line Tools to use nRF Util.
+    |nrf_CLT_deprecation_note|
+
+* Removed the Toolchain Manager app section from the following pages:
+
+  * :ref:`installation`
+  * :ref:`updating`
+  * :ref:`requirements`
+
+  The app has been deprecated and no longer provides the latest toolchain and |NCS| versions for installation.
+
+Board support
+=============
+
+* Removed support for the nRF52810 Desktop Mouse board (``nrf52810dmouse/nrf52810``).
+
+Build and configuration system
+==============================
+
+* Removed support for the deprecated multi-image builds (parent-child images) functionality.
+  All |NCS| projects must now use :ref:`sysbuild`.
+  See :ref:`child_parent_to_sysbuild_migration` for an overview of differences with parent-child image and how to migrate.
+* Updated:
+
+  * The default runner for the ``west flash`` command to `nRF Util`_ instead of ``nrfjprog`` that is part of the archived `nRF Command Line Tools`_.
+    For more information, see the :ref:`build system section in the v3.0.0 migration guide <migration_3.0_recommended>` and the :ref:`programming_selecting_runner` section on the programming page.
+  * Erasing the external memory when programming a new firmware image with ``west flash`` series now always correctly honors the ``--erase`` flag (and its absence) both when using the ``nrfjprog`` and ``nrfutil`` backends.
+    Prior to this release, the ``nrjfprog`` backend would always erase only the sectors of the external flash used by the new firmware, and the ``nrfutil`` backend would always erase the whole external flash.
+
+Developing with nRF70 Series
+============================
+
+* Added:
+
+  * Support for the nRF7002 EB II (PCA63571) with the nRF54 Series DKs as detailed in :ref:`ug_nrf7002eb2_gs`.
+  * A new section :ref:`ug_nrf70_wifi_enterprise_mode` in the :ref:`ug_nrf70_wifi_advanced_security_modes` page.
+
+* Deprecated support for the nRF7002 EB (PCA63561) with the nRF54 Series DKs.
+* Removed support for storing the nRF70 firmware patches in external flash without the :ref:`partition_manager`, as mentioned in :ref:`ug_nrf70_developing_fw_patch_ext_flash`.
+
+Developing with nRF54L Series
+=============================
+
+* Added:
+
+  * HMAC SHA-256 with a 128-bit key type to KMU, as detailed in the :ref:`ug_nrf54l_crypto_kmu_supported_key_types` documentation section.
+  * A workaround for nRF54L15 Errata 30.
+    Use ``CLOCK_CONTROL_NRF_HF_CALIBRATION=y`` to explicitly activate the workaround.
+    Consequence of activating the workaround might be increased power consumption due to periodic CPU wake-up, so use it only if errata conditions are met.
+    Note that the workaround is already included in MPSL component, so the solution is applicable needed only if your application does not utilize MPSL.
```

Comment:
> so the solution is applicable needed only if your application does not utilize MPSL.

Either `applicable` or `needed` should be probably removed. I guess `applicable` is more appropriate

Reply:
Fixed